lib/src/generator/generator_frontend.dart

```diff
@@ -34,6 +35,19 @@ class GeneratorFrontEnd implements Generator {
   /// Traverses the package graph and generates documentation for all contained
   /// elements.
   List<Indexable> _generateDocs(PackageGraph packageGraph, FileWriter writer) {
+    runtimeStats.accumulators['writtenCategoryFileCount'] = 0;
```

Comment:
This is a lot of repetitive code. We could add a method `resetAccumulators(Set<String> names)` in `_RuntimeStats`.

Reply:
Done